### PR TITLE
Tajaran Sleek Veil Flashproofing

### PR DESCRIPTION
**Pull request by Onyxarias**
## What Does This PR Do
Adds flash immunity to the Sleek Veil, bringing it up to spec with the standard Security HUD sunglasses. ~~Also adds species restriction to all of the veils, so that only Tajaran can wear them~~

## Why It's Good For The Game
Tajaran sec officers shouldn't have to nerf themselves in order to flavour their character with lore-specific gear. Choosing the sleek veil over a standard set of HUD sunglasses was always a disadvantage, and shouldn't be when every officer has access to them from roundstart. ~~There is no reason why another race should be able to spawn with veils, given the stated lore on them. Gives Taj a reason to follow their lore regarding veils.~~

## Changelog
:cl:
add: Adds the flash_protect ability to the Sleek Veil
/:cl:

### CODE REVIEW

**Comment by kenken244** [comment]
There shouldn't be a species restriction. Both because I want to wear it to offend cats and also because of balance. If an anyway takes out a tator, they need to be able to use the sec hud

**Comment by Aurorablade** [comment]
Species restriction is a no go i did attempt that when i ported them orignally and no. Its not something to be done, its fine without the restricts on it really.

**Comment by Onyxarias** [comment]
I'll wait to see what a maintainer has to say about it. If it's an issue, I can change it.

**Comment by Aurorablade** [comment]
Maintainer was the one who told me to change it when i orginally ported it. Trust me stance has not changed.

**Comment by Djvoid225** [comment]
lorewise flashproof veils are more than supported, and species restriction isn't really that big of a difference when it comes down to gameplay (though I can for sure see it messing with some newer antag players, so MAYBE cut that.) otherwise I see this is an absolute win. 

More lore and rp individuality enforced by gameplay. Thumbs up from me. (and the... two people that play Tajaran.)

**Comment by Wojz** [comment]
SecHUD that only tajs can use... that's not gonna fly. But I don't think the flashproof is a big deal tbh. Not like sunglasses are hard to comeby for anyone who selects those.

**Comment by ZanderDoggo** [comment]
I like the idea of it, and I think it would probably be better as "loadout" locked for other species than Taj. If anyone can still wear it but just can't roundstart with it, then I think it would be fine. It allows for antags and other players to still be able to steal a fully functioning sec hud without other species spawning with veils which doesn't really make much sense overall.

**Comment by S34NW** [comment]
I think adding the flash protection makes sense, but species-locking it? No thank you. Not to mention that non-tajarans can still have it as part of their loadout, but then be unable to wear it once in-game.

**Comment by Onyxarias** [comment]
Removed the species restriction. Until I can figure out how to make them only SPAWN on Taj, instead of being only wearable by them, they will remain available to all.;

**Review by SabreML — CHANGES_REQUESTED** [review]
image
Not *entirely* sure how you managed that, but you should be able to solve it by just deleting the `Paradise` folder in the PR branch and committing again.

(This one:)
image

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -171,6 +171,7 @@
 	desc = "An Ahdominian made veil that allows the user to see while obscuring their eyes. This one has an in-built security HUD."
 	icon_state = "tajblind_sec"
 	item_state = "tajblind_sec"
+	flash_protect = 1
```
Comment (Fox-McCloud) [inline code/modules/clothing/glasses/hud.dm:174]: Use the define please.

**Comment by Fox-McCloud** [comment]
Please fix the unnecessary file change so this can be merged.

**Comment by Onyxarias** [comment]
Oh stars, uh. I'm going to be honest, totally forgot about this.
I'm sorry for the messup, bear with me while I relearn things again and get it fixed.
Sorry ;^^

**Review by dearmochi — CHANGES_REQUESTED** [review]
The branch seems to include the repo as a sub module, which is probably not what you want to do

**Review by variableundefined — CHANGES_REQUESTED** [review]
This PR is approved and ready to go, except for the part where the repo is a submodule.

**Review by SabreML — APPROVED** [review]

File: code/modules/clothing/glasses/hud.dm (hunk 1 above)
Comment (Onyxarias) [reply code/modules/clothing/glasses/hud.dm]: Done and done, thank you. I used the standard sechud as a reference, which also doesn't use the define

**Pull request merged**